pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1313,27 +1319,40 @@ func setIOThreads(vmi *v1.VirtualMachineInstance, domain *api.Domain, vcpus uint
 	if !hasIOThreads(vmi) {
 		return
 	}
+	currentAutoThread := defaultIOThread
 	ioThreadCount, autoThreads := getIOThreadsCountType(vmi)
 	if ioThreadCount != 0 {
 		if domain.Spec.IOThreads == nil {
 			domain.Spec.IOThreads = &api.IOThreads{}
```

Comment:
random unrelated question. I could never figure out why kubevirt uses `<iothreads>` instead of the configurable `<iothreadids>`:
https://libvirt.org/formatdomain.html#iothreads-allocation

in this PRs context, is there any benefit in using the overloaded version?
I believe it's achievable without breaking the end kubevirt API user

Reply:
Not sure, I get the question. But we are using both. The 'iothreads' defines how many iothreads in total we have, while each iothreadid allows a specific assignment for each disk. In this case, we are assigning all the iothreads to all the disks which require iothread, but you could have different configuration.

Reply:
okay just summarizing offline discussion, we don't use iothreadids in kubevirt and it's not relevant to this PR